### DISCUSSION TITLE
Weapon upgrades

### DISCUSSION THREAD

**Discussion by johnny336hun** [OGSR/OGSR-Engine#354] [Q&A]
Hey guys,

I have bought the Remington kit from the merc trader in AW, but I cannot install it on the gun, receiving the "no available slot for this addon" overlay message.

No technician even offers installing it on my gun.

Tried searching for it on moddb, google, reddit, but couldn't find the answer.

Please, help me out!

Thanks!

**Comment by Kirito03** [comment]
You must have **short** Remington **without** addons (scopes, grips, etc.) **in** slot to able to replace barrel with the kit...
